test/org/usfirst/frc/team2585/tests/RobotMapTest.java

```diff
@@ -0,0 +1,19 @@
+package org.usfirst.frc.team2585.tests;
+
+import org.junit.*;
```

Comment:
Although this works, it is bad practice to import everything if you don't need it. There is a small(almost miniscule) increase in compilation time, however, the main reason to not use wildcard imports is that you should try to be as verbose as possible and show only what you need.